repository/src/main/java/org/alfresco/repo/content/filestore/FileContentStore.java

```diff
@@ -639,4 +640,10 @@ private void ensureFileInContentStore(File file)
             throw new ContentIOException("Access to files outside of content store root is not allowed: " + file);
         }
     }
+
+    @Override
+    public boolean isStorageClassesSupported(Set<String> storageClasses)

```

Comment:
We could consider having this at a level closer to the ContentStore hierarchy.

Reply:
Moved the code snippet, as discussed regarding `default` to the `ContentStore`